R/key-helpers.R

```diff
@@ -62,16 +62,20 @@ cdm_check_constraints <- function(dm) {
 check_key <- function(.data, ...) {
   data_q <- enquo(.data)
   .data <- eval_tidy(data_q)
-  args <- exprs(...)
-  names(args) <- set_names(paste0("...", seq_along(args)))
+
+  cols_avail <- colnames(.data)
+  cols_chosen <- tidyselect::vars_select(cols_avail, ...)
+  # if no column is chosen, all columns are used for the check
+  if (is_empty(cols_chosen)) cols_chosen <- cols_avail
```

Comment:
`if (missing(...))` ?

Reply:
that throws an error in case a tidyselect-helper function is used, but it does work with `if (is_empty(enexprs(...)))`, see new commits